1-exercises/B-object-arrays/exercise-1.js

```diff
@@ -13,8 +13,8 @@ let people = [
   { name: "Elise", age: 35 },
 ];
 
-let peopleNames =
-  /* todo */
+let peopleNames = [people[0].name, people[1].name, people[2].name];
```

Comment:
```suggestion
let peopleNames = people.map((person) => person.name);
```
your code works very well, but if you had more names you could use this code to display them all.